docs/complexity-antipatterns-and-refactoring-strategies.md

```
@@ -88,11 +88,11 @@ Cognitive Complexity is incremented based on three main rules 8:
    `try-catch` blocks, jumps to labels, and sequences of logical operators like
    `&&` and `||`), a penalty is applied.
 
-2. **Nesting:** Each level of nesting of these flow-breaking structures adds an
+1. **Nesting:** Each level of nesting of these flow-breaking structures adds an
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 1; Style: 1/2/3

(MD029, ol-prefix)